### PR TITLE
2014-02-12-sunhwacodingbatexercises2.md

### PR DESCRIPTION
**Pull request by sunhwap**
I have finished the second codingbat exercises, and it is found at the following link:  
http://sunhwap.github.io/spring2014/2014/02/12/sunhwacodingbatexercises2.html

I am also including herokupost link in this pull request, and it is found at the following link:
http://sunhwap.github.io/spring2014/02/21/sunhwaherokupost.html
However, it is not displayed since github pages is messing up for me.

I am adding a post about me link, and it is found at the following link:  
http://sunhwap.github.io/spring2014/2014/02/28/sunhwaabout.html
However, github pages is still messing up for me.

### CODE REVIEW

**Comment by gerbal** [comment]
@sunhwap please move the file into the _posts directory. Having a post in the root (spring2014) directory will break the site.

**Comment by sunhwap** [comment]
Hi Grant,I do not understand which file to which you are referring.  When I created the new post, I clicked the + sign next to post.   Could you tell me how to find this new post?Do you mean that I need to move this post in the post folder(directory)?Thanks.sunhwa

Date: Wed, 12 Feb 2014 12:02:31 -0800

**Comment by gerbal** [comment]
This one: 
https://github.com/sunhwap/spring2014/blob/gh-pages/2014-02-12-sunhwacodingbatexcercises2.md

**Comment by sunhwap** [comment]
Hi Grant,When I opened this link, either edit key or delete key is disabled,  could help me to find a way to edit?  I also added _post/ to the file name.  However, it is still not displayed.thanks.

Date: Wed, 12 Feb 2014 12:19:39 -0800

**Comment by gerbal** [comment]
Are you logged in to github?

On Feb 12, 2014 3:58 PM, sunhwap notifications@github.com wrote:
Hi Grant,When I opened this link, either edit key or delete key is disabled, could help me to find a way to edit? I also added _post/ to the file name. However, it is still not displayed.thanks.

Date: Wed, 12 Feb 2014 12:19:39 -0800

**Comment by sunhwap** [comment]
I signed in, and when I clicked Edit key, it does give me a while screen rather than the black screen.  so I can't commit changes.thanks.

Date: Wed, 12 Feb 2014 13:01:57 -0800

File: _posts/2014-02-12-sunhwacodingbatexercises2.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+
+---
+
```
Comment (eah13) [inline _posts/2014-02-12-sunhwacodingbatexercises2.md]: There are several empty lines here: lines one, three, and 8.  Something made you leave them empty but I think this is what's cusing the confusion.

File: _posts/2014-02-05-testtesty.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+author: sunhwap
+title: sunhwa's a very cool title for a test post
+layout: post
+date: 2014-02-19
+---
+
+## Hi !
+
+{ % include python %}
```
Comment (eah13) [inline _posts/2014-02-05-testtesty.md]: The space between the `{` and `%` is making this not render correctly.  Be attentive with your code!

File: _posts/2014-02-05-testtesty.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+print("I am Python code")
+
+{% include endpython %} #
```
Comment (eah13) [inline _posts/2014-02-05-testtesty.md]: There is an `#` on the end of this line.  I don't think it's messing anything up, but why is it here?  Every character counts.

**Comment by eah13** [comment]
Still a few issues with this pull request.  Please resolve them quickly.  Also, when you recieve these notifications, please come to github.com to respond to them instead of replying to the email: the email responses include the entire conversation each time..

**Pull request merged**